lib/models/situation.js

```diff
@@ -164,6 +164,7 @@ var LogementDef = {
     conjointMemeAdresse: Boolean,
     loyer: Number,
     adresse: AdresseDef,
+    threeOutOfFiveYears: Boolean,
```

Comment:
I don't find this name explicit enough. What do you think? What about something like `inhabitantForThreeYearsOutOfFiveLast`? It's ugly, but this is so specific the varname will likely have to be as specific…